crates/storage/src/blueprint.rs

```diff
@@ -111,6 +139,7 @@ where
     fn delete(storage: &mut S, key: &M::Key, column: S::Column) -> StorageResult<()>;
 }
 
+// TODO: Rename
```

Comment:
Could you add more description to the todo please?=D

Reply:
Oh missed this todo... In previous iterations I added another trait next to this, in which case it would be nice to rename this one. That's no longer needed. I'll remove the todo comment.

Reply:
Removed in https://github.com/FuelLabs/fuel-core/pull/2396/commits/8b35460e2c60b3de75e0d32f58928aca264884c5